pkg/istiod/pilotstart.go

```diff
@@ -300,7 +300,7 @@ func defaultMeshConfig() *meshconfig.MeshConfig {
 	meshConfigObj := mesh.DefaultMeshConfig()
 	meshConfig := &meshConfigObj
 
-	meshConfig.SdsUdsPath = "./var/run/sds/sds_path"
+	meshConfig.SdsUdsPath = "unix:/etc/istio/proxy/SDS"
```

Comment:
I think the reason is we want to differentiate the two paths. If so, can we make the path name saying the property, like "unix:/var/run/local_sds/sds_path"?

Reply:
The rootfs is read only - with the exception of /etc/istio/proxy, which is a mem-mounted writable dir.

And having 'path' in the name doesn't make sense - it is a path already. I can use lower case sds if it helps. 

Another option that is now possible is to use the abstract network namespace instead of files, but not sure it's worth it.